src/lib/Query/Content/CriterionVisitor/ContentName.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Solr\Query\Content\CriterionVisitor;
+
+use Ibexa\Contracts\Core\Repository\Values\Content\Query\Criterion;
+use Ibexa\Contracts\Solr\Query\CriterionVisitor;
+use QueryTranslator\Languages\Galach\Generators\ExtendedDisMax;
+use QueryTranslator\Languages\Galach\Parser;
+use QueryTranslator\Tokenizing;
+
+final class ContentName extends CriterionVisitor
```

Comment:
visitors should be internal

Reply:
https://github.com/ibexa/solr/pull/61/commits/11557906e7904effa39728ff7cb6b281aeab7a9e